docs/methods/cardanoSignTransaction.md

```diff
@@ -15,33 +15,37 @@ TrezorConnect.cardanoSignTransaction(params).then(function(result) {
 });
 ```
 
-### Params 
+### Params
 [****Optional common params****](commonParams.md)
-###### [flowtype](../../src/js/types/networks/cardano.js#L60-L179)
+###### [flowtype](../../src/js/types/networks/cardano.js#L87-L211)
 * `signingMode` - *obligatory* [CardanoTxSigningMode](#CardanoTxSigningMode)
-* `inputs` - *obligatory* `Array` of [CardanoInput](../../src/js/types/networks/cardano.js#L62)
-* `outputs` - *obligatory* `Array` of [CardanoOutput](../../src/js/types/networks/cardano.js#L77)
+* `inputs` - *obligatory* `Array` of [CardanoInput](../../src/js/types/networks/cardano.js#L87)
+* `outputs` - *obligatory* `Array` of [CardanoOutput](../../src/js/types/networks/cardano.js#L103)
 * `fee` - *obligatory* `String`
 * `protocolMagic` - *obligatory* `Integer` 764824073 for Mainnet, 42 for Testnet
 * `networkId` - *obligatory* `Integer` 1 for Mainnet, 0 for Testnet
 * `ttl` - *optional* `String`
 * `validityIntervalStart` - *optional* `String`
-* `certificates` - *optional* `Array` of [CardanoCertificate](../../src/js/types/networks/cardano.js#L124)
-* `withdrawals` - *optional* `Array` of [CardanoWithdrawal](../../src/js/types/networks/cardano.js#L131)
-* `auixiliaryData` - *optional* [CardanoAuxiliaryData](../../src/js/types/networks/cardano.js#143)
+* `certificates` - *optional* `Array` of [CardanoCertificate](../../src/js/types/networks/cardano.js#L150)
+* `withdrawals` - *optional* `Array` of [CardanoWithdrawal](../../src/js/types/networks/cardano.js#L158)
+* `auxiliaryData` - *optional* [CardanoAuxiliaryData](../../src/js/types/networks/cardano.js#L173)
+* `mint` - *optional* [CardanoMint](../../src/js/types/networks/cardano.js#L164)
+* `additionalWitnessRequests` - *optional* `Array` of `string | Array<number>` (paths). Used for multi-sig and token minting witness requests as those can not be determined from the transaction parameters.
 * `metadata` - *removed* - use `auxiliaryData` instead
 
 ### CardanoTxSigningMode
 
-[Type definition](../../src/js/types/trezor/protobuf.js#L78)
+[Type definition](../../src/js/types/trezor/protobuf.js#L98)
 
 #### `ORDINARY_TRANSACTION`
 
-Represents an ordinary user transaction transferring funds.
+Represents an ordinary user transaction transferring funds, delegating stake or withdrawing rewards. The transaction will be witnessed by keys derived from paths included in the inputs, certificates and withdrawals. Additionaly, if token minting is present, transaction will also be witnessed by keys derived from paths included in `additionalWitnessRequests`.
 
 The transaction
 - *should* have valid `path` property on all `inputs`
 - *must not* contain a pool registration certificate
+- *must* contain only 1852 and 1855 paths
+- *must* contain 1855 witness requests only when transaction contains token minting/burning
```

Comment:
A bit ambiguous, is this "if and only if"? The parties signing inputs and minting could be different.

The same for multisig.

Reply:
I agree. Maybe replacing `must contain only` with `may contain only` would fix this?

Reply:
There's no convention, the original docs have been copied from LedgerJS so that's where the style comes from. `may contain only` resolves this also in my opinion. Or we could switch this up to `*must not* contain 1855 witness requests when transaction is not minting/burning tokens`